index.bs

```diff
@@ -5119,12 +5117,10 @@ partial interface MLGraphBuilder {
   </summary>
   <div class=algorithm-steps>
     1. Let |outputShape| be an empty array of {{unsigned long}}.
-    1. If |newShape| is a scalar [=number=], set |outputShape| to the [=/list=] « 1 ».
-    1. Otherwise, if |newShape| is an array of {{unsigned long}}:
-        1. If the [=list/size=] of |newShape| is 0, set |outputShape| to the [=/list=] « 1 » (reshaping to scalar).
-        1. If any value in |newShape| is 0, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
-        1. Let |inputElementCount| be the product of all elements in |input|.{{MLOperand/[[descriptor]]}}.{{MLOperandDescriptor/dimensions}}.
-        1. If product of all values in |newShape| is not equal to |inputElementCount|, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
+    1. If the [=list/size=] of |newShape| is 0, set |outputShape| to the empty [=/list=] « » (reshaping to scalar).
```

Comment:
Per above, prefer "an empty list" and drop the « ».

Also the "(reshaping to scalar)" seems like a non-normative comment. Maybe make this an actual Note? (I don't feel strongly about this.)

Reply:
Fine with dropping the chevrons. Was following the precedent I saw elsewhere (e.g. `set it to the [=/list=] « 0, 0, 0, 0 »`).

Deparenthesizing to `set |outputShape| to an empty [=/list=] for a scalar`.